mule-extensions-api/src/main/java/org/mule/runtime/extension/api/ExtensionConstants.java

```diff
@@ -145,6 +145,22 @@
    */
   public static final String SCHEDULING_STRATEGY_PARAMETER_DESCRIPTION = "Configures the scheduler that triggers the polling";
 
+  /**
+   * The name of an attribute which allows declaring an error mapping
+   */
+  public static final String ERROR_MAPPINGS_PARAMETER_NAME = "errorMappings";
+
+  /**
+   * The description of the parameter for declaring an error mapping
+   */
+  public static final String ERROR_MAPPINGS_PARAMETER_DESCRIPTION = "Set of 'errorMapping's";
```

Comment:
remove single quotes. This is supposed to be human readable so it should just say `error mappings`